packages/kms-local/src/key-management-system.ts

```diff
@@ -257,14 +264,19 @@ export class KeyManagementSystem extends AbstractKeyManagementSystem {
    * @returns a `0x` prefixed hex string representing the signed digest in compact format
    */
   private eth_rawSign(managedKey: string, data: Uint8Array) {
-    return new SigningKey('0x' + managedKey).signDigest(data).compact
+    // Is compact serialized really what we want?
+    return new SigningKey('0x' + managedKey).sign(data).compactSerialized
```

Comment:
`compactSerialized` in v6 seems to be the equivalent of `compact` in v5, so it should work as before.

You are right to question it as we need to document things better.